styles/themes/theme.less

```diff
@@ -290,6 +290,33 @@ table.snapshottable {
     }
 }
 
+.stage-access(@color: black) {
+    display: inline-block;
+    min-width: 1.5em;
+    .center-align;
+    font-size: 1.1em;
+    font-weight: bold;
```

Comment:
I'd like to see all three of the access indicator characters slightly larger, if possible. The green checkmark isn't as heavy as the red x to begin with, so could stand to be somewhat larger, if possible. However, that would require passing a size parameter in the mixin, as well as the color. If it's a reasonable thing to do, I suggest 1.4em for the green checkmark, and 1.2 em for the other two. Otherwise, I'd suggest taking all of them to 1.2 instead of 1.1.

Reply:
I'm amenable to making them all 1.2em, but then I'd like help on figuring out how the @#$! to center the text contents within the cell. The Info icon already makes those cells taller than the others and the alignment is worse with the larger font size.`vertical-align: middle` isn't working and for the life of me I can't seem to figure it out.

Reply:
Oh. Well, bah.

Looking at PROD, though, the current icons aren't vertically centered, either: 
<img width="275" alt="image" src="https://user-images.githubusercontent.com/36800239/94099671-20f38f80-fdf1-11ea-80c9-197c2860926a.png">

Also, comparing it against the narrower version for narrow window/mobile devices, it looks like you're not slimming that column down to just the round ID? Was that deliberate?

Just looking at the code (and remember, my HTML is rusty), `$access_icon` is inside a `<div>` element, and the rest of the text in that table cell is outside that. (And yes, I realize that the div is `inline-block`. Instead of `vertical-align: middle`, have you tried shifting the baseline by a specific amount?

Reply:
> Looking at PROD, though, the current icons aren't vertically centered, either:
> <img alt="image" width="275" src="https://user-images.githubusercontent.com/36800239/94099671-20f38f80-fdf1-11ea-80c9-197c2860926a.png">

It's not the icon centering that I'm worried about -- it's the text.

> Also, comparing it against the narrower version for narrow window/mobile devices, it looks like you're not slimming that column down to just the round ID? Was that deliberate?

It was that time, yes.

Reply:
I've added a WIP (work-in-progress) commit to this MR to increase the font size and updated the sandbox. To best highlight the weird not-vertically-centered stuff look with an account that is eligible for an activity but not access to it (like one that has PP but not PPV).

Reply:
> I'm amenable to making them all 1.2em, but then I'd like help on figuring out how the @#$! to center the text contents within the cell. The Info icon already makes those cells taller than the others and the alignment is worse with the larger font size.`vertical-align: middle` isn't working and for the life of me I can't seem to figure it out.

Okay, with the increase in size to 1.2em, what I'm seeing is that, when the icon overlaps with the descriptive text, it causes an increase in line height for that top line, pushing the second line down:
<img width="132" alt="image" src="https://user-images.githubusercontent.com/36800239/94173853-1caf8c80-fe5a-11ea-9beb-3ef7ab7cb560.png">

But if it doesn't overlap, there's no change:
<img width="193" alt="image" src="https://user-images.githubusercontent.com/36800239/94173959-3e107880-fe5a-11ea-91f5-1666037bb2a2.png">

And if there's only one line, it's aligned to the middle based on the tallest line. Which is the icon?
<img width="253" alt="image" src="https://user-images.githubusercontent.com/36800239/94174819-63ea4d00-fe5b-11ea-974f-bc93318ff676.png">

Short of putting the icon in a separate table cell, is there a way to cause it to center on the normal text, and not have the icon cause perturbations?

So yeah, I see why you sized it at 1.1 initially. I'm willing to go back to that.

Reply:
> I've added a WIP (work-in-progress) commit to this MR to increase the font size and updated the sandbox. To best highlight the weird not-vertically-centered stuff look with an account that is eligible for an activity but not access to it (like one that has PP but not PPV).

Yes, that makes it even more obvious, though I did finally figure out what you meant, once you increased the size of the icon. Short of disassociating them from each other completely, I'm not sure I see a way to force the text to be centered vertically, independently of the icon.

Reply:
I moved it back down to 1.1em for now.